app/models/post.rb

```diff
@@ -1,17 +1,26 @@
 class Post < ApplicationRecord
+  validates :title, :text, presence: true
```

Comment:
We should also make a validation that checks for a max of 250 characters for the title. Please be sure to add this so that you meet the requirements 🤓

Reply:
The rest of the post validations are on point 🔥